plugins/amt/README.md

```diff
@@ -15,3 +15,12 @@ which can be found here: https://github.com/mjg59/mei-amt-check
 
 That tool in turn is heavily based on mei-amt-version from samples/mei in the
 Linux source tree and copyright Intel Corporation.
+
+GUID Generation
+---------------
+
+These devices use the standard USB DeviceInstanceId values, e.g.
+
+ * `USB\VID_1D50&PID_60C6&REV_0001`
+ * `USB\VID_1D50&PID_60C6`
+ * `USB\VID_1D50`
```

Comment:
Er, that doesn't seem right for AMT is it?

Reply:
`These devices use the existing GUID provided by the AMT host interface.`?

Reply:
Yeah that's better